tests/ulyanov_et_al_2016/test_data.py

```diff
@@ -1,20 +1,143 @@
+import itertools
+
 import pytest
 
-from pystiche_papers import ulyanov_et_al_2016 as paper
+import pytorch_testing_utils as ptu
+import torch
+from torch.utils.data import DataLoader
+
+from pystiche.data import ImageFolderDataset
+from pystiche.image import transforms
+from pystiche.image.transforms import CenterCrop, ValidRandomCrop
+from pystiche.image.transforms.functional import grayscale_to_fakegrayscale, resize
+from pystiche_papers.data.utils import FiniteCycleBatchSampler
+from pystiche_papers.ulyanov_et_al_2016 import data
+from pystiche_papers.utils import make_reproducible
 
 from .._asserts import assert_image_downloads_correctly, assert_image_is_downloadable
 
 
+def test_ulyanov_et_al_2016_content_transform(subtests):
+    make_reproducible()
+    image = torch.rand(1, 1, 32, 32)
+    edge_size = 16
+
+    for impl_params, instance_norm in itertools.product((True, False), (True, False)):
+        with subtests.test(impl_params=impl_params, instance_norm=instance_norm):
+            make_reproducible()
+            content_transform = data.ulyanov_et_al_2016_content_transform(
+                edge_size=edge_size,
+                impl_params=impl_params,
+                instance_norm=instance_norm,
+            )
+            actual = content_transform(image)
+
+            if impl_params:
+                if instance_norm:
+                    make_reproducible()
+                    transform = ValidRandomCrop(edge_size)
+                    transform_image = transform(image)
+                else:
+                    transform_image = resize(image, edge_size)
+            else:
+                transform = CenterCrop(edge_size)
+                transform_image = transform(image)
+
+            desired = grayscale_to_fakegrayscale(transform_image)
```

Comment:
I would split this off into a separate test, which test for this explicitly. The standard case should be RGB.